docs/src/how-to-use.md

```diff
@@ -337,3 +337,21 @@ Depending on the configuration of the energy storage assets, it may or may not b
+-   (optional) define a period partition in [`assets-timeframe-partitions.csv`](@ref schemas). If there is no partition defined, then by default the constraint is created for each period in the timeframe, otherwise, it will consider the partition definition in the file.
+
+For example, let's assume we have a year divided into 365 days because we are using days as periods in the representatives from the [_TulipaClustering.jl_](https://github.com/TulipaEnergy/TulipaClustering.jl). In addition, we define the `max_energy_timeframe_partition = 10 MWh`, meaning that the peak energy we want to have is 10MWh for each period or period partition. So, depending on the optional information we can have:
+
+-   _No definition of profile and period partitions_: The default profile is 1.p.u. for each period and since there are no period partitions then constraints will be for each period (i.e., daily). So, the outgoing energy of the asset for each day must be lower or equal to 10Mwh.
+-   _Profile definition, but no definition of period partitions_: The profile definition and value will be in the [`assets-timeframe-profiles.csv`](@ref schemas) and [`profiles-timeframe.csv`](@ref schemas) files. For example, we define a profile that has the following first four values: 0.6 p.u., 1.0 p.u., 0.8 p.u., and 0.4 p.u. Since there are no period partitions then constraints will be again for each period (i.e., daily). So, the outgoing energy of the asset for the first four days must be lower or equal to 6MWh, 10MWh, 8MWh, and 4Mh.
+-   _Profile and period partitions definition_: Same profile as before, but now we define a period partition in the [`assets-timeframe-partitions.csv`](@ref schemas) file as `uniform` with a value of 2. This value means that we will aggregate every two periods (i.e., every two days). So, instead of having 365 constraints, we will have 183 constraints (182 every two days and one last constraint of 1 day). Then the profile is aggregated with the sum of the values inside the periods within the partition. Thus, the outgoing energy of the asset for the first two partitions (i.e., every two days) must be lower or equal to 16MWh, and 12MWh.
+
+> **Note:**
+> If you want to set a limit on the maximum or minimum outgoing energy for a year with representative days, you can use the partition definition to create a single partition for the entire year to combine the profile. If you are using the default value for the profiles, which is 1.0 p.u., please note that the parameters `max_energy_timeframe_partition` and `min_energy_timeframe_partition` are defined per period, which means they represent daily limits. This implies that the energy limit for the whole year will be 365 times `max_energy_timeframe_partition` or 365 times `min_energy_timeframe_partition`.
```

Comment:
This is two different notes, right? I would split it, so people don't skip the second note if the first doesn't apply to them..

Reply:
Correct! I will split them. Thanks!